### ISSUE TITLE
Documentation bug? : Quickstart for SolidJS

### ISSUE THREAD

**Issue by hmatsu47** [supabase/supabase#6807]
# Improve documentation

## Link 

Add a link to the page which needs improvement (if relevant)
https://supabase.com/docs/guides/with-solidjs

## Describe the problem

Are the `supabaseUrl` and `supabaseAnonKey` descriptions in `src/supabaseClient.jsx` incorrect?

## Describe the improvement

```jsx:src/supabaseClient.jsx
import { createClient } from '@supabase/supabase-js'

const supabaseUrl = import.meta.env.VITE_SUPABASE_URL
const supabaseAnonKey = import.meta.env.VITE_SUPABASE_ANON_KEY

export const supabase = createClient(supabaseUrl, supabaseAnonKey)
```

(same as Vue 3)

## Additional context

Add any other context or screenshots that help clarify your question.
<img width="686" alt="image" src="https://user-images.githubusercontent.com/35142774/168080908-e9a4a029-cddf-4a69-b60c-372fcce02bb5.png">

**Comment by hmatsu47** [comment]
In addition, `className` and `htmlFor` have been deprecated in solidJS 1.4.0 and will need to be replaced with `class` and `for` in the future.

https://github.com/solidjs/solid/releases/tag/v1.4.0

**Comment by Hallidayo** [comment]
I've created a pr (#6893) for the `.env` values.

I did notice that the guide is doing a clone of [this solid-js template](https://github.com/solidjs/templates/tree/master/js) which is using version 1.3.13 of `solid-js` so the `className` and `htmlFor` should be okay but as you said @hmatsu47 these will need to be changed if the template uses version 1.4.0 and above.

**Comment by alaister** [comment]
Closed via #6893 

I was considering leaving this open because of `className` and `htmlFor`, however, since they still work I think they should have their own issue/pr. Contributions always welcome! 😊
